server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -137,6 +146,7 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
         checkpointManager: IDeliCheckpointManager,
         private readonly forwardProducer: IProducer,
         private readonly reverseProducer: IProducer,
+        private readonly isNewDocument: boolean,
```

Comment:
You shouldn't need a new argument for this. You can check `this.sequenceNumber === 0` instead